docs/customization.md

```diff
@@ -1,37 +1,9 @@
-            'ExcludeList' = @('5.6','5.7','5.10','5.11','5.14','5.28','5.32','5.40')
-            '2315AccountsRenameadministratoraccount' = 'CISAdmin'
-            '2316AccountsRenameguestaccount' = 'CISGuest'
-            '2375LegalNoticeText' = 'Legal says only we can use these machines.'
-            '2376LegalNoticeCaption' = 'Legal Notice'
-        }
-    }
-}
-```
+Every resource contains an 'ExcludeList' property that allows you to exclude settings based on their recommendation ID. These IDs can be found in the CIS documentation available from [CIS](./cis.md). Using these documents you should be able to identify your setting in question and locate its ID number. Examples of excluding recommendations can be found in the applicable [resources documentation](/src/CISDSC/docs).
 
 # How do I customize values of recommendations?
-Some recommendations are for ranges of values not an explicit value. Examples include minimum number of days or log size. These customizations are also accounted for in these resources via parameters. Available parameters can always be found with the below command substituting the resource name for your applicable OS.
-
-```
-(Get-DscResource -Module CISDSC -Name CIS_Microsoft_Windows_10_Enterprise_Release_1909).Properties
-```
-
-This will return something resembling the below screenshot. The naming convention is as follows: '111Enforcepasswordhistory' configures "1.1.1 (L1) Ensure 'Enforce password history' is set to '24 or more
-password(s)'". You can use the documentation linked above to identify the accepted values for this setting. The above code example also shows how to specify a parameter.
-</br>![Example](screenshots/parameters.PNG)
+Some recommendations are for ranges of values not an explicit value. Examples include minimum number of days or log size. These customizations are also accounted for in these resources via parameters. Available parameters and syntax can always be found in the [resources documentation](/src/CISDSC/docs).
 
 # Special cases
-There are a handful of special cases for customization. Those are displayed above for the legal disclaimer text and built-in account renames. These will require a value be provided as shown or their recommendation IDs be added to the ExclusionList. This is due to the fact that these fields are always organization specific so there are no safe defaults for these values.
\ No newline at end of file
+There are a handful of special cases for customization. Those are displayed above for the legal disclaimer text and built-in account renames. These will require a value be provided as shown or their recommendation IDs be added to the ExclusionList. This is due to the fact that these fields are always organization specific so there are no safe defaults for these values. More information on this can be found in the applicable [resources documentation](/src/CISDSC/docs).
```

Comment:
It says that these special cases "are displayed above". But since you took out the example, this isn't the case any more.

Reply:
Good catch! Should be fixed in latest push